ldi-core/ldes-client-v2/tree-node-supplier/src/main/java/ldes/client/treenodesupplier/repository/sqlite/EntityManagerFactory.java

```diff
@@ -0,0 +1,47 @@
+package ldes.client.treenodesupplier.repository.sqlite;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+
+public class EntityManagerFactory {
```

Comment:
This is again in my opinion a too ambiguous naming.. This hide an underlying class with the same name.

Reply:
Well, it's actually to enforce singleton use of the underlying class.